R/derive_var_dthcaus.R

```diff
@@ -147,12 +165,12 @@ derive_var_dthcaus <- function(dataset, ...) {
     ) %>%
     select(-starts_with("temp_"))
 
-  left_join(dataset, dataset_add, by = "USUBJID")
+  left_join(dataset_name, dataset_add, by = "USUBJID")
 }
 
 #' Create an `dthcaus_source` object
 #'
-#' @param dataset A data.frame containing a source dataset.
+#' @param dataset_name A data.frame containing a source dataset.
```

Comment:
Please update the description. You can use a similar description as in `tte_source()`.